catalyst_voices/packages/internal/catalyst_voices_services/lib/generated/catalyst_gateway/client_index.dart

```diff
@@ -1 +1 @@
-export 'cat_gateway_api.swagger.dart' show CatGatewayApi;
+export 'vit.swagger.dart' show Vit;
```

Comment:
What does happen with the cat_gateway_api? We will need it as well

Reply:
yeah, forgot about it.